### PR TITLE
added grommet to website

### PR DESCRIPTION
**Pull request by abhishekp106**
### List of Changes:

- Grommet was installed
- `<React.StrictMode>` component was changed to `<Grommet plain>` in `src/index.tsx` (Don't think this should raise any issues, but let me know!)
- Not Found page has some random boxes. Feel free to delete, Grommet should be relatively simple to use.
- Ping me if any weird errors!

### Notes:

- if `npm install` doesn't work, do `npm install grommet styled-components --save `
- We may want to redesign some of our stuff (The navigation bar can be written tbh (sorry Neil?), since it can now be responsive and is much simpler to maintain or update).

### Screencast and Images:

Screen Shot 2021-02-19 at 5 05 11 PM

### CODE REVIEW

**Comment by zpChris** [comment]
Ah, I totally forgot about this PR! Will review tonight and let you merge, sorry about that 👍

File: frontend/src/views/Landing.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ function LandingPage() {
       <TextBlockLink to="/Projects">Projects</TextBlockLink>
       <TextBlockLink to="/Stories">Stories</TextBlockLink>
       <TextBlockLink to="/Team">Team</TextBlockLink>
+      <TextBlockLink to="/Testing">Testing</TextBlockLink>
```
Comment (zpChris) [inline frontend/src/views/Landing.tsx]: Did you mean to add this? If so, what's it for, and did you mean to add `/Testing` as an actual page in some regard?

**Review by zpChris — APPROVED** [review]
LGTM! Before you merge though, I left a comment / nit on the "Testing" link on the `Landing.tsx` page - let me know if you meant to add that.

File: frontend/src/views/Landing.tsx (hunk 1 above)
Comment (abhishek-pandya) [reply frontend/src/views/Landing.tsx]: Nope! Let me get rid of it. Thanks for catching that!

**Pull request merged**